project/app/components/Footer.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import Icon from '@availity/icon';
 import AvLink from '@availity/link';
+import { useAppStore } from '@/hooks';
 
-const Footer = ({ toggleFlippable }) => (
-  <p className="text-center mt-3">
-    Made with <Icon title="Click to See Animated Card" name="heart" color="danger" onClick={toggleFlippable} />{' '}
-    by{' '}
-    <AvLink url="https://www.availity.com" target="_blank">
-      Availity
-    </AvLink>{' '}
-    2019
-  </p>
-);
+const Footer = () => {
+  const { toggleIsFlippable } = useAppStore(store => ({ toggleIsFlippable: store.toggleIsFlippable }));
```

Comment:
Does this need to be wrapped in observer in order for it to accept changes?

Not a crazy issue, just not great if someone comes here looking for an example and forgets to wrap it.

Reply:
Nope, the `useStore` hook provides the `useObserver` which will look for updates